source/_includes/common-tasks/network_storage.md

```diff
@@ -0,0 +1,81 @@
+## Network storage
+
+You can configure both Network File Share (NFS) and Windows samba (CIFS) targets to be used within Home Assistant and add-ons.
```

Comment:
```suggestion
You can configure both network file share (NFS) and Windows samba (CIFS) targets to be used within Home Assistant and add-ons.
```
use [lower case for spelled-out acronyms](https://learn.microsoft.com/en-us/style-guide/acronyms#lowercase-the-spelled-out-term)

Reply:
haha. You got me there. The MS Style Guide instructs to generally lowercase the spelled-out version of acronyms. Unless it is a proper noun. In this case, it seems to be a [proper noun](https://learn.microsoft.com/en-us/windows-server/storage/nfs/nfs-overview). I should have checked for the specific case rather than applying the general rule.